tsconfig.shared.json

```diff
@@ -5,6 +5,6 @@
     "declaration": true,
     "esModuleInterop": true,
     "strict": true,
-    "moduleResolution": "node"
+    "moduleResolution": "bundler"
```

Comment:
Since we bundle our code for the consumers of this library, "bundler" is what we need. It uses the module resolution algorithm of node16 without requiring the file extensions.

Reply:
Swapped this back to "node" to minimise changes to 6.5.0. Was able to get it to build ok, though will see how we go with a mock publish.